experimental/dds/tree2/src/test/shared-tree/fuzz/operationTypes.ts

```diff
@@ -111,6 +111,20 @@ export interface NodeRangePath {
 	count: number;
 }
 
+export interface DisconnectOp {
+	type: "disconnect";
+	index: number;
```

Comment:
Why is this index needed here? Can't we use the index from `TreeSelectionContext`? Same for `ReconnectOp.index`.